ipa-core/src/net/client/mod.rs

```diff
@@ -369,7 +391,8 @@ impl MpcHelperClient {
         gate: &Gate,
         data: S,
     ) -> Result<ResponseFuture, Error> {
-        let body = hyper::Body::wrap_stream::<_, _, Error>(data.map(Ok));
+        let data = data.map(|v| Ok::<bytes::Bytes, Error>(Bytes::from(v)));
+        let body = axum::body::Body::from_stream(data);
```

Comment:
I commented on a similar code down below, but I think we should not mix and match body combinators from Hyper and Axum. It would make it much harder to support both in our code. I'd prefer if we use hyper for everything related to url/request/body management and axum strictly for web server functionality (handlers/middleware)

Reply:
I'll investigate what a switch to Hyper body would entail. BTW, that body is defined on a separate more general http-body crate. The Axum body had a few functions that were useful, that why I was using it. It made the AxumBodyWrapper simpler.